src/plugins/procmon/procmon.cpp

```diff
@@ -145,6 +145,39 @@ struct process_visitor_ctx
 
 } // namespace
 
+static char* readcmdline(vmi_instance_t vmi, drakvuf_trap_info_t* info, addr_t addr)
```

Comment:
Maybe `read_command_line` would better match the naming convention? (The rest of functions is using names with underscores).